modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/classic_richardson.h

```diff
@@ -0,0 +1,65 @@
+// SPDX-FileCopyrightText: 2024 The OpenSn Authors <https://open-sn.github.io/opensn/>
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "modules/linear_boltzmann_solvers/lbs_solver/iterative_methods/wgs_context.h"
+#include "modules/linear_boltzmann_solvers/lbs_solver/groupset/lbs_groupset.h"
+#include "framework/math/linear_solver/linear_solver.h"
+#include <memory>
+#include <vector>
+#include <functional>
+
+namespace opensn
+{
+namespace lbs
+{
+
+/**Linear Solver specialization for Within GroupSet (WGS) solves with classic
+ * Richardson.
+ */
+class ClassicRichardson : public LinearSolver
+{
+public:
+  /**
+   * Constructor.
+   * \param gs_context_ptr Context Pointer to abstract context.
+   */
+  explicit ClassicRichardson(std::shared_ptr<WGSContext> gs_context_ptr);
+
+  ~ClassicRichardson() override;
+
+  void Setup() override {}
+
+  void Solve() override;
+
+protected:
+  void PreSetupCallback() override{};
```

Comment:
The `;` is not needed and is most likely causing this weird formatting. See line 32 for comparison. 

So remove the `;` and make the same change for the functions below as well.

Reply:
Looks like I got a bit `;` happy. Fixed!